src/data/Data.js

```diff
@@ -13,45 +13,38 @@ Data.accepts = jQuery.acceptData;
 
 Data.prototype = {
 
-	register: function( owner ) {
-		var value = {};
-
-		// If it is a node unlikely to be stringify-ed or looped over
-		// use plain assignment
-		if ( owner.nodeType ) {
-			owner[ this.expando ] = value;
-
-		// Otherwise secure it in a non-enumerable, non-writable property
-		// configurability must be true to allow the property to be
-		// deleted with the delete operator
-		} else {
-			Object.defineProperty( owner, this.expando, {
-				value: value,
-				writable: true,
-				configurable: true
-			});
-		}
-		return owner[ this.expando ];
-	},
 	cache: function( owner ) {
+		// Check if the owner object already has a cache
+		var value = owner[ this.expando ];
+		if ( value ) {
+			return value;
+		}
+
+		// If not, create one
+		value = {};
```

Comment:
I believe it was but I'll have to double check. I think I tried a few different methods of doing this and found this to be smallest (after gzip).